res/css/views/rooms/_RoomHeader.pcss

```diff
@@ -191,7 +191,7 @@ limitations under the License.
     }
 
     &:hover {
-        background: rgba($accent, 0.1);
+        background: $accent-300;
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes